hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -752,6 +755,33 @@ public void stop() {
     LOG.info("Container Balancer stopped successfully.");
   }
 
+  private void validateConfiguration(ContainerBalancerConfiguration conf) {
+    // maxSizeEnteringTarget and maxSizeLeavingSource should by default be
+    // greater than container size
+    long size = (long) ozoneConfiguration.getStorageSize(
+        ScmConfigKeys.OZONE_SCM_CONTAINER_SIZE,
+        ScmConfigKeys.OZONE_SCM_CONTAINER_SIZE_DEFAULT, StorageUnit.BYTES) +
+        OzoneConsts.GB;
+
+    if (conf.getMaxSizeEnteringTarget() < size) {
```

Comment:
This should be the check if another GB isn't added to `size` (as commented above)
```suggestion
    if (conf.getMaxSizeEnteringTarget() <= size) {
```